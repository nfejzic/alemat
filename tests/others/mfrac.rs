use alemat::{
    elements::{Frac, Ident},
    MathMl,
};

#[test]
fn frac() {
    let out = MathMl::with_content(
        Frac::builder()
            .num(Ident::from("x"))
            .denom(Ident::from("y"))
            .build(),
    )
    .render();

    crate::snap_test!(out, name: "others_frac");
}
