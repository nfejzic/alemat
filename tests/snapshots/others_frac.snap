---
source: tests/others/mfrac.rs
expression: out
---
<math><mfrac><mi>x</mi><mi>y</mi></mfrac></math>
